src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -322,6 +322,49 @@ function trapBubbledEventsLocal() {
         ),
       ];
       break;
+    case 'video':
+    case 'audio':
+      var mediaEvents = {
```

Comment:
Ah, no I meant to put it outside the function since it's a static object, so it isn't recreated every time it's called.